bothub/api/v2/evaluate/filters.py

```diff
@@ -139,6 +145,9 @@ def filter_repository_version(self, queryset, name, value):
 
     def filter_repository_cross_validation(self, queryset, name, value):
         return queryset.filter(cross_validation=value)
+    
```

Comment:
this line contains whitespace:

for more details: https://www.flake8rules.com/rules/W293.html